tests/SixLabors.ruleset

```diff
@@ -5,6 +5,8 @@
     <!-- Test only ignores -->
     <Rule Id="SA0001" Action="None" />
     <Rule Id="SA1117" Action="None" />
+    <Rule Id="SA1201" Action="None" />
```

Comment:
I'd love to get a tests version of our ruleset added to the Standards repo. I think it's very obvious now in ImageSharp the difference in readability when rules are dictated.